library/MedEx/API.php

```diff
@@ -2964,28 +2964,28 @@ public function MedExBank($stage)
                                     <div id="pwd_help" class="nodisplay signup_help"><?php echo xlt('Secure Password Required') . ": " . xlt('8-12 characters long, including at least one upper case letter, one lower case letter, one number, one special character and no common strings'); ?>...</div>
                                 </div>
                                 <div class="form-group mt-3">
                                     <label for="new_rpassword"><?php echo xlt('Repeat'); ?>:</label>
                                     <i id="pwd_rcheck" name="pwd_rcheck" class="top_right_corner nodisplay text-success fa fa-check"></i>
-                                    <input type="password" placeholder="<?php echo xla('Repeat password'); ?>" id="new_rpassword" name="new_rpassword" class="form-control" required>
+                                    <input type="password" placeholder="<?php echo xla('Repeat password'); ?>" id="new_rpassword" name="new_rpassword" class="form-control" required />
                                     <div id="pwd_rhelp" class="nodisplay signup_help" style=""><?php echo xlt('Passwords do not match.'); ?></div>
                                 </div>
                             </div>
                             <div id="ihvread" name="ihvread" class="text-left showReminders">
-                                            <input type="checkbox" class="updated required" name="TERMS_yes" id="TERMS_yes" required>
+                                            <input type="checkbox" class="updated required" name="TERMS_yes" id="TERMS_yes" required />
                                             <label for="TERMS_yes" class="input-helper input-helper--checkbox" data-toggle="tooltip" data-placement="auto" title="Terms and Conditions"><?php echo xlt('I have read and my practice agrees to the'); ?>
                                                 <a href="#" onclick="cascwin('https://medexbank.com/cart/upload/index.php?route=information/information&information_id=5','TERMS',800, 600);">MedEx <?php echo xlt('Terms and Conditions'); ?></a></label><br />
-                                            <input type="checkbox" class="updated required" name="BusAgree_yes" id="BusAgree_yes" required>
+                                            <input type="checkbox" class="updated required" name="BusAgree_yes" id="BusAgree_yes" required />
                                             <label for="BusAgree_yes" class="input-helper input-helper--checkbox" data-toggle="tooltip" data-placement="auto" title="BAA"><?php echo xlt('I have read and accept the'); ?>
                                             <a href="#" onclick="cascwin('https://medexbank.com/cart/upload/index.php?route=information/information&information_id=8','Bus Assoc Agree',800, 600);">MedEx <?php echo xlt('Business Associate Agreement'); ?></a></label>
                                             <br />
```

Comment:
The register button below this has incorrect classes - there shouldn't be a row with no classes